index.html

```diff
@@ -0,0 +1,484 @@
+<!doctype html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Avengers | Creative Design Agency</title>
+    <meta name="description" content="Avengers, a Creative Design Agency based out of San Fransisco California, speicalizing in Web Design, UI Design, Icon Design, Branding & Logotype">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <link rel="stylesheet" href="css/main.css">
+  </head>
+  <body>
+    <input type="checkbox" id="mobileNavToggle" class="mobileNavToggle">
+    <header class="l-nav">
+      <nav class="mainNav">
+        <ul role="navigation">
+          <li role="menuitem" class="nav-link"><a href="#home">Home.</a></li>
+          <li role="menuitem" class="nav-link"><a href="#services">Services.</a></li>
+          <li role="menuitem" class="nav-link"><a href="#portfolio">Portfolio.</a></li>
+          <li role="menuitem" class="nav-link"><a href="#about">About.</a></li>
+          <li role="menuitem" class="nav-link"><a href="#team">Team.</a></li>
+          <li role="menuitem" class="nav-link"><a href="#contact">Contact.</a></li>
+        </ul>
+      </nav>
+    </header>
+    <main class="page-wrapper" id="home">
+      <label for="mobileNavToggle" class="nav-toggle"></label>
+      <label for="mobileNavToggle" class="trigger"></label>
```

Comment:
I know this goes hand-in-hand with your off-canvas solution, but it's strange. You end up with these mystery labels in your markup (not anywhere near the input to which they're bound) and on top of that you have two that are attached to the same input (`mobileNavToggle`) but do completely different things. The more I'm seeing of this particular solution, the more it feels like a hack.